ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/JsonCallParameterUtil.java

```diff
@@ -36,9 +42,14 @@ public static JsonCallParameter validateAndGetCallParams(final JsonRpcRequestCon
     if (callParams.getGasPrice() != null
         && (callParams.getMaxFeePerGas().isPresent()
             || callParams.getMaxPriorityFeePerGas().isPresent())) {
-      throw new InvalidJsonRpcParameters(
-          "gasPrice cannot be used with maxFeePerGas or maxPriorityFeePerGas",
-          RpcErrorType.INVALID_GAS_PRICE_PARAMS);
+      try {
+        LOG.warn(
```

Comment:
@siladu do you think this log will be too spammy?

Reply:
hmmm good point maybe is, happy to switch to debug for now to avoid devnet noise.